app/src/main/java/org/wikipedia/WikipediaApp.java

```diff
@@ -381,6 +382,8 @@ public synchronized void resetWikiSite() {
     public void logOut() {
         L.d("Logging out");
         AccountUtil.removeAccount();
+        WikipediaApp.getInstance().getBus().post(new LoginLogoutEvent());
```

Comment:
Not sure but do we really need a `Bus` event to update the Nav tab view? Please see `loginLogoutClick()` in `MainActivity`